src/Accordion.php

```diff
@@ -0,0 +1,164 @@
+ *
+ * You can add static content to an accordion item or pass a callback
+ * for adding content dynamically.
+ */
+class Accordion extends View
+{
+    public $defaultTemplate = 'accordion.html';
+
+    public $ui = 'accordion';
+
+    /**
+     * The css class for Fomantic-ui accordion type.
+     *
+     * @var arra|string|ynull
+     */
+    public $type = null;
+
+    /**
+     * Settings as per Fomantic-ui accordion settings.
+     *
+     * @var array
+     */
+    public $settings = [];
+
+    /**
+     * A collection of AccordionItem in this Accordion;.
+     *
+     * @var array
+     */
+    public $items = [];
+
+    /**
+     * The AccordionItem index number to activate on load.
+     *
+     * @var int
+     */
+    public $activeItem = -1;
+
+    /**
+     * Add an accordion item.
+     * You can add static View within your item or pass
+     * a callback for dynamic content.
+     *
+     * @param string        $title
+     * @param null|callable $callback
+     * @param string        $icon
+     *
+     * @throws Exception
+     *
+     * @return View
+     */
+    public function addItem($title, $callback = null, $icon = 'dropdown')
+    {
+        $item = $this->add(['AccordionItem', 'title' => $title, 'icon' => $icon]);
+
+        if ($callback) {
+            $item->virtualPage = $item->add(['VirtualPage', 'ui' => '']);
+            $item->virtualPage->set($callback);
+        }
+
+        $this->items[] = $item;
+
+        return $item;
+    }
+
+    /**
+     * Activate or open an accordion item.
+     *
+     * @param AccordionItem $item The item to activate.
+     */
+    public function activate($item)
+    {
+        $this->activeItem = $this->getItemIdx($item);
+    }
+
+    /*
+     * JS Behavior wrapper functions.
+     */
+    public function jsRefresh($when = null)
+    {
+        return $this->jsBehavior('refresh', [], $when);
+    }
+
+    public function jsOpen($item, $when = null)
+    {
+        return $this->jsBehavior('open', [$this->getItemIdx($item)], $when);
+    }
+
+    public function jsCloseOthers($when = null)
+    {
+        return $this->jsBehavior('close others', [], $when);
+    }
+
+    public function jsClose($item, $when = null)
+    {
+        return $this->jsBehavior('close', [$this->getItemIdx($item)], $when);
+    }
+
+    public function jsToggle($item, $when = null)
+    {
+        return $this->jsBehavior('toggle', [$this->getItemIdx($item)], $when);
+    }
+
+    /**
+     * Return an accordion js behavior command
+     * as in Semantic-ui behavior for Accordion module.
+     * Ex: toggle an accordion from it's index value.
+     * $accordion->jsBehavior('toggle', 1).
+     *
+     * @param string $behavior The name of the behavior for the module.
+     * @param array  $args     The behaviors argument as an array.
+     * @param bool   $when     When this js action is render.
+     *
+     * @return mixed
+     */
+    public function jsBehavior($behavior, $args, $when = null)
```

Comment:
this is pretty cool functionality... perhaps this could be used across other views too ? Just a thought for now, we can refactor after.